### DISCUSSION TITLE
How to add nltk download to docker image?

### DISCUSSION THREAD

**Discussion by thejaminator** [bentoml/BentoML#1432] [Q&A]
Hi, I am serving a model that requires nltk.download("bunch of stuff").
Traditionally I would add these to my Dockerfile, so that it is present in my dockerimage. I run my image off google cloud run.
```
RUN python3 -m nltk.downloader punkt
RUN python3 -m nltk.downloader wordnet
RUN python3 -m nltk.downloader stopwords
```

Is there a way I can make the generated Dockerfile from bentoml always have these lines? Or is the best way to write a shell script to edit the Dockerfile for this.

**Comment by parano** [comment]
Hi @thejaminator  - I think this is what you are looking for:

```
@env(
setup_sh="""
python -m nltk.downloader punkt
python -m nltk.downloader wordnet
python -m nltk.downloader stopwords
"""
)
class MyPredictionService(BentoService):
    ...
```
